nuxeo-features/rest-api/nuxeo-rest-api-test/src/test/java/org/nuxeo/ecm/restapi/test/BatchUploadFixture.java

```diff
@@ -210,6 +214,8 @@ private void itCanUseBatchUpload(boolean noDrop) throws IOException {
             }
         }
 
+        txFeature.nextTransaction(); // TODO check with efge
```

Comment:
Have you checked?

Reply:
No, it was mainly a remember for the review.

Reply:
Looks good to me, after a REST call we must make sure we use the new values now visible by switching to a new transaction.